corehq/apps/export/static/export/js/datasource_export.js

```diff
@@ -0,0 +1,26 @@
+hqDefine("export/js/datasource_export",[
+    'jquery',
+    'knockout',
+], function ($, ko) {
+
+    function datasourceExportViewModel() {
+        'use strict';
+        var self = {};
+
+        self.dataSource = ko.observable();
+        self.haveDatasources = ko.computed(function () {
+            if (self.dataSource() === undefined) { return false; }
+            return self.dataSource().length > 0;
```

Comment:
It is fine as is, but this could all be put in a single line as `return self.datasource() !== undefined && self.dataSource().length();`.